pagesTest/residents/[id]/index.spec.tsx

```diff
@@ -66,4 +66,19 @@ describe('ResidentPage', () => {
 
     expect(screen.getByText('This resident has no case notes yet.'));
   });
+
+  it("doesn't let normal users change restricted status", () => {
```

Comment:
"Doesn't let unauthorised users change restricted status"

Also I feel that there should be a test showing the opposite case, that a user with permission is able to make a change